test/integration/vaultauthmethods/terraform/main.tf

```
@@ -169,6 +169,81 @@ path "auth/${vault_auth_backend.approle.path}/login" {
 EOT
 }
 
+# aws auth config
+resource "kubernetes_service_account" "irsa_assumable" {
+  metadata {
```

Super nit, but I think you can put a `count` on this resource too?
```suggestion
    count = var.run_aws_tests || run_aws_static_creds_test? 1 : 0
    metadata {
```

---

Yeah, I think run_aws_tests should suffice since the static creds test doesn't use irsa. Updated in 1383a3b